pkg/kubectl/describe.go

```diff
@@ -2699,7 +2699,6 @@ func printLabelsMultilineWithIndent(w *PrefixWriter, initialIndent, title, inner
 			w.Write(LEVEL_0, "%s", innerIndent)
 		}
 		w.Write(LEVEL_0, "%s=%s\n", key, labels[key])
-		i++
```

Comment:
i'm curios, is it really work? anyway for loop works for every elements, and i is not used.
```
func main() {
	inList := []string{"first", "second", "third", "fourth"}
	for i, v := range inList {
		fmt.Println(i, v)
		i++
	}
}
```
output
```
0 first
1 second
2 third
3 fourth
```